outputs/jira.go

```diff
@@ -19,6 +19,11 @@ import (
 	"github.com/aquasecurity/go-jira"
 )
 
+const (
+	defaultIssueType     = "Task"
+	defaultIssuePriority = "High"
```

Comment:
If it's a non English JIRA, what would happen? Would the first priority be selected from the returned list?

Reply:
we can't get an analogue of the `high` priority from [rest API](https://github.com/aquasecurity/postee/pull/271/files#diff-76265623be2e5ca53abae86b5d1087ff8bcdc391272004877039fe54435b1f38R312-R319).
If there is no `defaultIssuePriority` in the priority list we will use [the first priority from the getting list](https://github.com/aquasecurity/postee/pull/271/files#diff-76265623be2e5ca53abae86b5d1087ff8bcdc391272004877039fe54435b1f38R312-R323).
User can change the priority in the configuration file, if he needs it.

Reply:
Roger that